include/RAJA/policy/desul/atomic.hpp

```diff
@@ -26,6 +26,27 @@ using raja_default_desul_scope = desul::MemoryScopeDevice;
 namespace RAJA
 {
 
+RAJA_SUPPRESS_HD_WARN
+template <typename AtomicPolicy, typename T>
+RAJA_HOST_DEVICE
+RAJA_INLINE T
+atomicLoad(AtomicPolicy, T volatile *acc) {
+  return desul::atomic_load(const_cast<T*>(acc),
+                            raja_default_desul_order{},
+                            raja_default_desul_scope{});
+}
+
+RAJA_SUPPRESS_HD_WARN
+template <typename AtomicPolicy, typename T>
+RAJA_HOST_DEVICE
+RAJA_INLINE void
+atomicStore(AtomicPolicy, T volatile *acc, T value) {
```

Comment:
```suggestion
atomicStore(AtomicPolicy, T volatile *acc, T value)
{
```

Format consistency with other routines.

Reply:
Fixed.